packages/enzyme-test-suite/package.json

```diff
@@ -49,4 +49,4 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.6.1"
   }
-}
+}
```

Comment:
please revert this; all files should have a trailing newline

Reply:
sorry i'm not sure why my editor keeps doing this, it otherwise is set to add trailing nls